### DISCUSSION TITLE
【灌水】以太适应自测结果是这个分数，我能去空洞里搬砖吗？

### DISCUSSION THREAD

**Discussion by niecod** [share121/inter-knot#147] [其他]
本人学生，高中还没毕业。老爸因为工作原因买了以太适应自测卡，我出于好奇就顺便也跟着测了一下，结果拿了个80分。老爸当时就从椅子上跳了起来，逼着我重新测了一次…据说50分能从事空洞相关工作的及格线，80分是很高的分数了，这是不是意味着我也可以去空洞里当搬运工了？
![QQ20240809-192726](https://github.com/user-attachments/assets/1bfae023-410e-405a-bbfc-c449b1349690)

**Comment by wushanef** [comment]
这边建议当绳匠呢，工资高（虽然不稳定)，但还可以做其他兼职

**Comment by Luli-Chen** [comment]
绳匠有被抓风险呢…还是优先考虑稳定工作吧

**Comment by SDCOM-0415** [comment]
这边建议去**零号空洞**和我一起打工！

**Comment by March-777th** [comment]
夺少？！80分！要不然猜猜对空六科和特勤局哪个先联系你

**Comment by shishenyihe** [comment]
一楼谁家小孩
